### PR TITLE
#5 'Homepage custom fields'

### PR DESCRIPTION
**Pull request by SmruthyJoseph**
#8 **Homepage custom fields** issue resolved.

### CODE REVIEW

**Review by AbhijithHaridas — CHANGES_REQUESTED** [review]

File: wp-content/themes/soraytec/functions.php [hunk 1]
```diff
@@ -715,48 +715,7 @@ function signup() {
     }
     add_action( 'init', 'signup' );
 
-    // custom post type for Footer  for About page
-
-
-function footer() {
```
[the remark this answers is not in the document]
Comment (SmruthyJoseph) [reply wp-content/themes/soraytec/functions.php:721]: I customized footer using acf first so the function is for that and  after i changed to customizer , so i deleted the function.

[the remark this answers is not in the document]
Comment (AbhijithHaridas) [reply wp-content/themes/soraytec/functions.php:721]: Not in the scope of this issue

[the remark this answers is not in the document]
Comment (SmruthyJoseph) [reply wp-content/themes/soraytec/functions.php:721]: Ok i will reset it.

**Comment by SmruthyJoseph** [comment]
#8 **Home page custom fields** issue corrected.

**Pull request merged**